dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/afus/AfusTransformationLog.scala

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.monster.dap.afus
```

Comment:
🤔 These classes feel like a copy and paste of the HLES classes [here](https://github.com/DataBiosphere/dog-aging-ingest/blob/master/dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/hles/HLESurveyTransformationLog.scala). Any reason those can't be named something more generic and reused rather than duping here? Perhaps as simple as dropping the `HLES` prefix on those classes and putting in a non-HLES package?